System.IO.FileSystem/File.cs

```diff
@@ -13,90 +14,65 @@ namespace System.IO
     /// </summary>
     public static class File
     {
-        #region Constants
-
-        private const int _defaultCopyBufferSize = 2048;
-
-        #endregion
-
-
-        #region Static Methods
+        private const int ChunkSize = 2048;
+        private static readonly byte[] EmptyBytes = new byte[0];
 
         /// <summary>
         /// Copies an existing file to a new file. Overwriting a file of the same name is not allowed.
         /// </summary>
         /// <param name="sourceFileName">The file to copy.</param>
         /// <param name="destFileName">The name of the destination file. This cannot be a directory or an existing file.</param>
-        /// <exception cref="ArgumentException">sourceFileName or destFileName is null or empty</exception>
-        public static void Copy(
-            string sourceFileName,
-            string destFileName)
+        /// <exception cref="ArgumentException"><paramref name="sourceFileName"/> or <paramref name="destFileName"/> is null or empty.</exception>
+        public static void Copy(string sourceFileName, string destFileName)
         {
-            Copy(sourceFileName, destFileName, false);
+            Copy(sourceFileName, destFileName, overwrite: false);
         }
 
         /// <summary>
         /// Copies an existing file to a new file. Overwriting a file of the same name is allowed.
         /// </summary>
         /// <param name="sourceFileName">The file to copy.</param>
         /// <param name="destFileName">The name of the destination file. This cannot be a directory.</param>
-        /// <param name="overwrite">true if the destination file can be overwritten; otherwise, false.</param>
-        /// <exception cref="ArgumentException">sourceFileName or destFileName is null or empty</exception>
-        public static void Copy(
-            string sourceFileName,
-            string destFileName,
-            bool overwrite)
+        /// <param name="overwrite"><c>true</c>; if the destination file can be overwritten; otherwise, <c>false</c>.</param>

```

Comment:
```suggestion
        /// <param name="overwrite"><see langword="true"/> if the destination file can be overwritten; otherwise, <see langword="false"/>.</param>

```

Reply:
You'll call me a pain.... 😅

Reply:
I always forget about langword but never forget that <c> makes it bold just the same ;)